content/about/notes/0.8/index.md

```diff
@@ -8,7 +8,10 @@ This is a major release for Istio on the road to 1.0. There are a great many new
 
 ## Networking
 
-- **Revamped Traffic Management Model**. We're finally ready to take the wraps off our [new traffic management APIs](/blog/2018/v1alpha3-routing/). We believe this new model is easier to understand while covering more real world deployment [use-cases](/docs/tasks/traffic-management/). For folks upgrading from earlier releases there is a [migration guide](/docs/setup/kubernetes/upgrading-istio/) and a conversion tool built into `istioctl` to help convert your config from the old model.
+- **Revamped Traffic Management Model**. We're finally ready to take the wraps off our
```

Comment:
well this sentence violates the best practices - but as your cleaning up the .spelling file, this wfm :)